tools/scripts/build-c-shared-linux.sh

```
@@ -16,22 +16,78 @@ replace_escaped=$(echo "$replace" | sed 's/[\/&]/\\&/g')
-rm -f build/libdefradb.so build/libdefradb.h
+echo "Removing existing build artifacts..."
+rm -rf build
+mkdir -p build
 
 echo "Building shared object..."
-CGO_ENABLED=1 GOARCH=amd64 GOOS=linux go build -tags "cshared ${BUILD_TAGS:-}" $BUILD_FLAGS \
-  -buildmode=c-shared -o build/libdefradb.so ./cbindings
+CGO_ENABLED=1 GOARCH=amd64 GOOS=linux \
+go build \
+  -tags "cshared ${BUILD_TAGS:-}" \
+  $BUILD_FLAGS \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)